nightscout/config.json

```diff
@@ -5,29 +5,31 @@
   "description": "Nightscout acts as a web-based CGM (Continuous Glucose Montinor) to allow multiple caregivers to remotely view a patients glucose data in realtime.",
   "url": "https://github.com/marciogranzotto/addon-nightscout",
   "arch": ["aarch64", "amd64"],
-  "webui": "[PROTO:ssl]://[HOST]:[PORT:5423]",
   "startup": "services",
   "init": "false",
   "boot": "auto",
-  "image": "marciogranzotto/image-{arch}-addon-nightscout",
   "hassio_role": "default",
+  "ingress": "true",
   "ports": {
-    "5423/tcp": 5423
+    "80/tcp": null
```

Comment:
Won't this make the add-on ONLY available through ingress? Which url are you supposed to use to upload entries from xdrip?

Reply:
I've a working add-on on my local installation, it works both with ingress and with direct connection.

I should check if my PR is updated and reflect my current working version

Reply:
I've compared with my master branch and I fixed this issue